### ISSUE TITLE
Dropdown's children should hover over Modal when expanded

### ISSUE THREAD

**Issue by Magneticmagnum** [DigitalRiver/react-atlas#395]
Or modal should expand?  Currently Modal will create a scrollbar, but it is bad UX

Example Styleguidist code:
```javascript
initialState = {active: false}
handleToggle = () => {
    setState({ active: !state.active })
};
<div> 
    <Button primary onClick={handleToggle}>Open Modal</Button>
    <Modal 
        active={state.active} 
        onOverlayClick={handleToggle} 
        onEscKeyDown={handleToggle}
        title="Modal Example Title"
    >
      <Dropdown>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
        	<span> 1 </span>
      </Dropdown>
    </Modal>
</div>
```

**Comment by Magneticmagnum** [comment]
Example image:
![image](https://user-images.githubusercontent.com/1256043/31892051-a924b57c-b7cd-11e7-94f0-37baf8c16214.png)

**Comment by hbowden** [comment]
Fixed by #397
